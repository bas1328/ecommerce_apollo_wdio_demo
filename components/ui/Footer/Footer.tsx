import React from 'react'
 import styles from './Footer.module.scss'

export default function Footer() {
  return (
    <footer className={styles.container}>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Pariatur reprehenderit aspernatur odit quisquam. Ipsam dolores aliquam omnis consequatur ipsa, excepturi vitae nisi illum repellendus molestiae odio itaque molestias mollitia laborum dicta corrupti quae vel quas dignissimos doloribus. Laudantium, autem! Nobis dolor, porro voluptatem alias quod beatae atque adipisci quas. Natus.</footer>
  )
}
