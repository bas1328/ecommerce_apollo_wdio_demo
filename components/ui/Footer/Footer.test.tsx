import { render, screen } from "@testing-library/react";
import Footer from "./Footer";
import "@testing-library/jest-dom";

describe("Footer", () => {
  it("renders a Footer", () => {
    const screen = render(<Footer />);

    const heading = screen.getByText(/lorem/i);

    expect(heading).toBeInTheDocument();
    expect(screen).toMatchSnapshot();
  });
});
