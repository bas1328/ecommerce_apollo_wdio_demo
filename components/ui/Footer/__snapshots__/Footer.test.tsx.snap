// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders a Footer 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="container"
      >
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Pariatur reprehenderit aspernatur odit quisquam. Ipsam dolores aliquam omnis consequatur ipsa, excepturi vitae nisi illum repellendus molestiae odio itaque molestias mollitia laborum dicta corrupti quae vel quas dignissimos doloribus. Laudantium, autem! Nobis dolor, porro voluptatem alias quod beatae atque adipisci quas. Natus.
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="container"
    >
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Pariatur reprehenderit aspernatur odit quisquam. Ipsam dolores aliquam omnis consequatur ipsa, excepturi vitae nisi illum repellendus molestiae odio itaque molestias mollitia laborum dicta corrupti quae vel quas dignissimos doloribus. Laudantium, autem! Nobis dolor, porro voluptatem alias quod beatae atque adipisci quas. Natus.
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
